server/verification-request/verification-request.service.ts

```diff
@@ -64,6 +67,22 @@ export class VerificationRequestService {
             query["topics.label"] = { $in: topics };
         }
 
+        if (severity && severity !== "all") {
```

Comment:
todo: if we don't already have a enum for the severity levels, let's create one and use it